docs/api_reference/subscriptions_api/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Subscriptions API",
```

Comment:
Maybe this should be documented inside the webhook section? Or at least referenced in the intro (e.g. there's also a programmatic way of doing this -> Link to this part)

Reply:
Yes good idea, I can reference it in the webhooks guide too 👍🏼